tezos/encoding-derive/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tezos_encoding_derive"
+version = "0.1.0"
```

Comment:
@akoptelov 
please, change version according to other modules, actual on develop is `version = "1.5.0"`

Reply:
Done